src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-weight: 300;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
```

Comment:
Remove redundant default values. Fix it everywhere

Reply:
./ How to add to git?

Reply:
just remove `font-style: normal;` it is already applied as default value